modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/test/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValidatorTest.java

```diff
@@ -160,13 +160,13 @@ public void testDuplicateFieldName() throws Exception {
 			createAvailableLocales(LocaleUtil.US), LocaleUtil.US);
 
 		ddmForm.addDDMFormField(
-			new DDMFormField("Name1", DDMFormFieldType.TEXT));
+			_createDDMFormField("Name1", DDMFormFieldType.TEXT, "Reference1"));
 
-		DDMFormField name2DDMFormField = new DDMFormField(
-			"Name2", DDMFormFieldType.TEXT);
+		DDMFormField name2DDMFormField = _createDDMFormField(
+			"Name2", DDMFormFieldType.TEXT, "Reference2");
```

Comment:
@devpedrotavares can you rename `name2DDMFormField` to `ddmFormField`?

Reply:
Done!